pyaedt/edb_core/edb_data/padstacks_data.py

```diff
@@ -1071,11 +1071,12 @@ def in_polygon(self, polygon_data, include_partial=True, simple_check=False):
     @property
     def component(self):
         """Get component this padstack belong to."""
```

Comment:
```suggestion
        """Get the component that this padstack belong to."""
```
Improving readability.